README.md

```diff
@@ -1,8 +1,13 @@
 # Zivid ROS driver
 
-This is the official ROS driver for [Zivid 3D cameras](https://www.zivid.com/).
+**This is a preview of the ROS 2 driver for Zivid cameras.**
 
-_A preview branch with support for ROS 2 is [now available here](https://github.com/zivid/zivid-ros/tree/ros2-preview)!_
+_Thank you for testing this preview of our ROS2 driver! Be aware that there may be adjustments to behavior and API_
+_before the official production release, which is scheduled for July._
+
+_Please read the rest of this README.md for a description of the driver, and see [CHANGELOG.md](./CHANGELOG.md) for noteworthy changes._
+
+_Contact us via issue https://github.com/zivid/zivid-ros/issues/50, or via customersuccess@zivid.com, to provide feedback or report issues._
```

Comment:
We must update this. Also point to ros1-master for ROS1 code.

Reply:
b11482c9b92690a4085c2f7c24589c0b05cc1a15